test/integration/helpers/setup.js

```diff
@@ -0,0 +1,20 @@
+const startupTasks = require('~/src/startup-tasks')
```

Comment:
FYI - this file doesn't get run as a regular test. Ava ignores all directories named `helpers` by default. A bad choice imo, but since exclude patterns don't work, may as well take advantage of it for test helpers.

See issue/code:
* https://github.com/avajs/ava/blob/87eef846392d04b727b57981a16f7e8429a04195/lib/ava-files.js#L76
* https://github.com/avajs/ava/issues/909
* https://github.com/avajs/ava/blob/bdc57f4f9d69d6998fa7b764590b8e4d1f0fbe3a/test/watcher.js#L585

Reply:
Yeah I agree that it was a bad decision. Definitely a random "gotcha" that adds little value.